torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -236,12 +236,35 @@ struct GraphFuser {
     return true;
   }
 
+  // XXX: this is O(n) where n is the number of outputs of the FusionGroup
+  bool isFusedConcatOutput(Value * producer) {
+    JIT_ASSERT(producer->node()->kind() == prim::FusionGroup);
+    auto * fusion_group = producer->node();
+
+    // Find the output index
+    auto outputs = fusion_group->outputs();
+    auto it = std::find(outputs.begin(), outputs.end(), producer);
+    JIT_ASSERT(it != outputs.end());
+    int64_t output_index = it - outputs.begin();
```

Comment:
Is it really the case that we can both concatenated and regular outputs from a fusion group? Can we have multiple concats fused too?

Reply:
Also, `output_index` is simply`producer->offset()`.